src/boards/include/boards/wiznet_w5100s_evb_pico.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+
+
+#ifndef _BOARDS_WIZNET_W5100S_EVB_PICO_H
+#define _BOARDS_WIZNET_W5100S_EVB_PICO_H
+
+// For board detection
+#define WIZNET_W5100S_EVB_PICO
+// --- BOARD SPECIFIC ---
+#ifndef W5100S_EVB_PICO_INTN_PIN
+#define W5100S_EVB_PICO_INTN_PIN 20
+#endif
+
+#ifndef W5100S_EVB_PICO_RSTN_PIN
+#define W5100S_EVB_PICO_RSTN_PIN 21
+#endif
+
+#ifndef W5100S_EVB_PICO_LED_PIN
+#define W5100S_EVB_PICO_LED_PIN 25
+#endif
+
+#ifndef W5100S_EVB_PICO_ADC_VREF_PIN
+#define W5100S_EVB_PICO_ADC_VREF_PIN 29
+#endif
+
+#ifndef W5100S_EVB_PICO_A0_PIN
+#define W5100S_EVB_PICO_A0_PIN 26
+#endif
+#ifndef W5100S_EVB_PICO_A1_PIN
+#define W5100S_EVB_PICO_A1_PIN 27
+#endif
+#ifndef W5100S_EVB_PICO_A2_PIN
+#define W5100S_EVB_PICO_A2_PIN 28
+#endif
```

Comment:
I don't think there's any point adding defines for these, because these GPIO numbers are "hardwired" on RP2040 and can't be changed on _any_ board.

Reply:
> 

I saw for something similar.
(https://github.com/raspberrypi/pico-sdk/blob/master/src/boards/include/boards/pimoroni_tiny2040.h )

Pin **W5100S_EVB_PICO_ADC_VREF_PIN**  will be deleted as it is judged that it can be deleted.

Reply:
Oh interesting.
Although digging into it some more, I see that https://shop.pimoroni.com/products/tiny-2040 has "A0", "A1" and "A2" silkscreened onto the PCB, which is probably why they chose to add defines for those to the header file? Whereas for your board it probably just makes sense to stick to the GPIO numbers, i.e. no extra defines for these pins? :shrug: